docker-compose.integration.yaml

```diff
@@ -17,7 +17,7 @@ services:
     networks:
       kafka:
   local-kafka1:
-    image: confluentinc/cp-kafka:5.5.5
+    image: confluentinc/cp-kafka:7.3.2
```

Comment:
This file is supposed to be synced from .github as mentioned in the comment above. Can we follow the same?